README.md

```diff
@@ -143,22 +130,23 @@ KarhooApi.authService.login(token = "eyJra...zRQ7Tn3A").execute {
 }
 ```
 
-It is also possible to verify issued tokens
-by using [the token exchange endpoint directly from our API explorer](https://developer.karhoo.com/reference#post_oauth-v2-token-exchange).
+It is also possible to authenticate with issued tokens using [the token exchange endpoint directly from our API explorer](https://developer.karhoo.com/reference#post_oauth-v2-token-exchange).
 
 ## 4. Implementing it in your backend
 
-Finally, once you have successfully authenticated a user in Karhoo SDK using the tokens you issued, check out the examples on how both the JWKS endpoint and token generation can be implemented in your code.
+Finally, once you have successfully authenticated a user using Karhoo SDK and the tokens you issued, check out the examples on how both the JWKS endpoint and token generation can be implemented in your code.
```

Comment:
Is this part correct: `once you have successfully authenticated a user using Karhoo SDK and the tokens you issued`?